### ISSUE TITLE
Selected resolution option is always the maximum (bottom) entry

### ISSUE THREAD

**Issue by Wedge009** [wesnoth/wesnoth#1772]
In Preferences -> Display -> Resolution, the maximum available resolution is always shown as selected instead of the actual current resolution.

Current master, 1.13.8+dev, df6a775e. Found this on Windows and Linux, seems to have been introduced fairly recently.

**Comment by Vultraz** [comment]
Likely causes are ed9c4dff35 and its fixup e188379cd2.

**Comment by Vultraz** [comment]
Though on my machine the last one is the correct option.

**Comment by CelticMinstrel** [comment]
That's a pretty dumb statement. It's the correct option if you don't change it, maybe.

**Comment by Wedge009** [comment]
Oh, I just noticed Vultraz's commits above. I noticed it's fixed in AR branch, but I still seem to get the problem in master. Unless I need to do a full rebuild. :/

**Comment by CelticMinstrel** [comment]
Should this be reopened?

**Comment by Wedge009** [comment]
Any reason for reviewing closed issues? I did fresh builds within the last day or so - 1.14 is fine, master is broken but in a different way to originally described here (the selection is 'correct' but the listed resolutions are wrong). Though as far as I know I understand master is very much still unstable.

**Comment by jostephd** [comment]
@Wedge009 Issues that were fixed on old master (and thus were closed) but are not fixed on new master should be reopened.

**Comment by jostephd** [comment]
@Wedge009 Yes. The master branch was copied to https://github.com/wesnoth/wesnoth/tree/accelerated-rendering-attempt / https://github.com/wesnoth/wesnoth/tree/old_master_20181015 and a new branch (#3603) was force-pushed to become new master. This was done because (old) master was unplayable (#3538). Issues #3614 and #3622 are consequent cleanups. There was discussion on https://forums.wesnoth.org/viewtopic.php?f=16&t=48837 (developers-only link).

Bottom line, master should be playable currently, though it will be some time before we have worked through the backlog of fwdports.

**Comment by Wedge009** [comment]
Thanks for the links. Especially that forum discussion, I see a lot of passionate opinions. I'm sorry for being even less involved than the little I usually am and missed all of that discussion.

I suppose this also explains the flurry of activity I've been seeing on 'haldric' project recently. I understand there's still no consensus but given what I've tried to skim-read there is simultaneous work on restoring master to a working state (looks done) and people working on the 2.0 prototype and/or 1.14/1.15 development according to their preferences.

Anyway, all that aside, I confirm (new) master is working with respect to this issue, it can remain closed.
